models/_sector/nft/trades/solana/platforms/magiceden_solana_trades.sql

```diff
@@ -0,0 +1,248 @@
+{{
+    config(
+        schema = 'magiceden_solana'
+        , tags = ['dunesql']
+        , alias = alias('trades')
+        ,materialized = 'incremental'
+        ,file_format = 'delta'
+        ,incremental_strategy = 'merge'
```

Comment:
can we also add incremental predicates to the materializations? it can really help during incremental runs

Reply:
Idk what this means, where and what do you want me to put

Reply:
not sure what this means/looks like

Reply:
Add `incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],` in the config here. This will make it so that the merge into also filters on the destination table instead of doing a full scan which improves performance when the destination table is big.